docs/source/conf.py

```diff
@@ -87,7 +94,7 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-html_title = "TraitsUI 4 User Manual"
+html_title = "TraitsUI 5 User Manual"
```

Comment:
i can do some string formatting and use the version information from `execfile(...)`. would that be okay?